### PR TITLE
REF: WalletButton to TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qy7o4ls7x5xbl6loy3dcc3oboe?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/975)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: components/WalletButton.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  testID?: string;
+  onPress: () => void;
+  size: {
+    width: DimensionValue | undefined;
+    height: DimensionValue | undefined;
+  };
+  active: boolean;
+}
+
+// Define the button details. Adjust image paths as necessary
```
Comment (Overtorment) [inline components/WalletButton.tsx:24]: comment

File: components/WalletButton.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+const buttonDetails: Record<string, ButtonDetails> = {
+  Bitcoin: {
+    image: require('../img/addWallet/bitcoin.png'), // Adjust path as necessary
```
Comment (Overtorment) [inline components/WalletButton.tsx:27]: comment

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xpsrqjolwtk7h2bnhlykpp5jfq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/986)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/j6lttx2b264gmhcxusmfbrymoe?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1043)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by Overtorment — CHANGES_REQUESTED** [review]
are you keeping it just for lulz..?

image

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ot5d5lmcvvcoupdwqwktifuj4i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1058)

**Pull request merged**